Bottom: 93c07ff797e3d0c6e229fbb98fd9430e8a20b78c
Top:    8b7f20545a233ec930ffd0b00fae0fe6bd660a5e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-26 22:44:34 +0100

Refresh of patch

---

diff --git a/src/data.h b/src/data.h
index 06daf0c..470933b 100644
--- a/src/data.h
+++ b/src/data.h
@@ -49,6 +49,8 @@ extern PCBType *PCB;
 #define max_copper_layer (PCB->Data->LayerN)
 #define bottom_silk_layer (max_copper_layer + BOTTOM_LAYER)
 #define top_silk_layer (max_copper_layer + TOP_LAYER)
+#define bottom_soldermask_layer (max_copper_layer + BOTTOM_SOLDERMASK_LAYER)
+#define top_soldermask_layer (max_copper_layer + TOP_SOLDERMASK_LAYER)
 
 extern SettingType Settings;
